packages/scripts/config/.eslintrc.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
+	root: true,
```

Comment:
Is this a bug worth fixing separately, lest it be stalled otherwise by the progress of the other changes in the pull request?

Reply:
Well, I'm not sure if it is a bug. It might be. I figured out it's safer to put it here :)
I will open another PR.

Reply:
#13483 is ready